bindings/netstandard/ElectionGuard/ElectionGuard.Encryption/ElectionGuard.Encryption.csproj

```diff
@@ -5,7 +5,7 @@
     <Platforms>x64;x86</Platforms>
 
     <!-- Project -->
-    <RootNamespace>ElectionGuard.Encryption</RootNamespace>
+    <RootNamespace>ElectionGuard</RootNamespace>

```

Comment:
This was originally labeled this way because the package was supposed to be **only** handling Encryption. I understand the refactor for this but to keep in mind the historical context.

Reply:
I acknowledge the historical context.  However, when this property and the actual namespaces are out of sync it produces linting errors in every single file.  Why not fix those linting errors?

Reply:
This should definitely be fixed. Merely making note.